lib/kbsecret/session.rb

```diff
@@ -80,6 +80,16 @@ def add_record(type, label, *args)
       record.sync!
     end
 
+    # Import an existing record
+    # @param record [Record] an instance of a record
+    # @return [void]
+    def import_record(record)
+      klass = record.class
+      new_record = klass.new(self, record.label, record.data.values.first)
```

Comment:
Is this invocation correct? The `initialize` method for records takes a trailing hash containing non-internal fields, not the first `data` value.

To avoid overwriting those internal fields (used on things like `todo` records), it might be better to use `load!` here:

```ruby
imported_record = klass.load! self, record.to_h
records << imported_record
imported_record.sync!
```

(FD: I haven't tested the code above).